.github/workflows/build-ros2-ws-action.yml

```diff
@@ -0,0 +1,88 @@
+name: ros2-ws-action
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master, feature/fix-github-workflows]
+  workflow_run:
+    workflows: [Build and Test Nightly]
+    types:  [requested, completed, in_progress]
+    branches: [feature/fix-github-workflows]
```

Comment:
I don't know why we need this, but it seems that having this branch to be triggered by the `pull_request` event is enough.
```suggestion
```